src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function showNotification(message, cls) {
+  const div = document.createElement('div');
+  const body = document.querySelector('body');
+
+  div.classList.add(cls);
+  div.textContent = message;
+  body.append(div);
+};
+
+const promise1 = Promise.resolve(() => {});
+const promise2 = Promise.reject(new Error());
```

Comment:
this promise should be rejected after 3 seconds and you reject it right away. You just show message in 3 seconds, but the promise is rejected right away

Reply:
@qoa11a done